packages/datadog-plugin-cucumber/test/index.spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+
+const {
+  TEST_FRAMEWORK,
+  TEST_TYPE,
+  TEST_NAME,
+  TEST_SUITE,
+  TEST_STATUS
+} = require('../../dd-trace/src/plugins/util/test')
+const { expect } = require('chai')
+const path = require('path')
+const { PassThrough } = require('stream')
+
+const TESTS = [
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':3',
+    requireName: 'simple.js',
+    testName: 'pass scenario',
+    status: 'pass',
+    success: true
+  },
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':8',
+    requireName: 'simple.js',
+    testName: 'fail scenario',
+    status: 'fail',
+    success: false
+  },
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':13',
+    requireName: 'simple.js',
+    testName: 'skip scenario',
+    status: 'skip',
+    success: true
+  },
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':19',
+    requireName: 'simple.js',
+    testName: 'skip scenario based on tag',
+    status: 'skip',
+    success: true
+  }
+]
+
+wrapIt()
+
+describe('Plugin', () => {
+  let Cucumber
+
+  withVersions(plugin, '@cucumber/cucumber', version => {
+    afterEach(() => {
+      // > If you want to run tests multiple times, you may need to clear Node's require cache
+      // before subsequent calls in whichever manner best suits your needs.
```

Comment:
I think this was only needed for `mocha`. Have you tried without it and it doesn't work?

Reply:
cucumber was marking steps as not implemented.